components/ApplicationProviders/ApplicationProviders.tsx

```diff
@@ -95,13 +96,13 @@ export const ApplicationProviders: FunctionComponent = ({ children }) => {
           <CenterProvider
             network={centerNetwork as any}
             apiKey={process.env.NEXT_PUBLIC_CENTER_KEY ?? ''}>
-            <TargetProvider>
-              <PaprBalanceProvider>
-                <TimestampProvider>
+            <TimestampProvider>
```

Comment:
now that target relies on timestamp, we need `TimestampProvider` to be outside `TargetProvider`